pkgs/development/libraries/libgtkflow/default.nix

```diff
@@ -0,0 +1,39 @@
+{stdenv, lib, vala, meson, ninja, pkg-config, fetchgit, gobject-introspection, glib, gtk3}:
+
+stdenv.mkDerivation rec {
+  pname = "libgtkflow";
+  version = "0.8.0";
+
+  src = fetchgit {
+    url = "https://notabug.org/grindhold/libgtkflow";
+    rev = "${version}";
+    hash = "sha256:1m30rvj5hx3b4cj8lbzrxv4j8lp3hx4jlb8vpf4rh46vc1rdkxpz";
+  };
+
+  nativeBuildInputs = [
+    vala
+    meson
+    ninja
+    pkg-config
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    gtk3
+    glib
+  ];
+
+  mesonFlags = [
+    "-Denable_valadoc=true"
+  ];
+
+  meta = with lib; {
+    description = "Flow graph widget for Gtk3";
+    homepage = "https://notabug.org/grindhold/libgtkflow";
+    maintainers = with maintainers; [ grindhold ];
+    license = licenses.lgpl3;
```

Comment:
`lgpl3` is unclear, use `lgpl3Plus` or `lgpl3Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

Reply:
done